src/helpers/sync/SyncTimeline.cpp

```diff
@@ -6,7 +6,16 @@
 #include <sys/eventfd.h>
 using namespace Hyprutils::OS;
 
+static bool checkDrmSyncobjTimelineSupport(int drmFD) {
+    uint64_t cap = 0;
+    int ret = drmGetCap(drmFD, DRM_CAP_SYNCOBJ_TIMELINE, &cap);
+    return (ret == 0 && cap != 0);
```

Comment:
this should probably be static. It won't change, and I don't know how expensive drmGetCap is. Furthermore, you're duplicating this here and in protocolmanager. Maybe add a bool to CCompositor that is set in `::initServer`?